Samples/Sample.Xamarin.Core/ViewModels/MainPageViewModel.cs

```diff
@@ -28,6 +28,12 @@ public MainPageViewModel()
             UnhandledCmd = new Command(Unhandle);
             BrokenViewCmd = new Command(GotoBrokenView);
             FeedbackCmd = new Command(ShowFeedback);
+            NativeCrashCmd = new Command(NativeCrash);
+            try
+            {
+                _nativeCrashService = DependencyService.Get<INativeCrash>();
+            }
+            finally { }
```

Comment:
why an empty finally block?

Reply:
To ignore  Platforms that didn't implement INativeCrash.

```csharp
        private Action NativeCrash => () =>
        {
            _nativeCrashService?.BrokenNativeCallback();
            DisplayAlert("Whoops", "A native crash sample wasn't implemented for this platform, mind opening a pull request? :)", "Yes");
        };
```

The Native Crash will not crash and an alert will be shown to the user explaining it.

Reply:
shou;dn't it be a catch block then?

Reply:
whoops, it seems like in that case, it doesn't throw an exception> Removing the try/finally block.